ports/servo/glutin_app/keyutils.rs

```diff
@@ -131,129 +131,132 @@ pub fn char_to_script_key(c: char) -> Option<constellation_msg::Key> {
-        VirtualKeyCode::Numpad9 => Ok(Key::Kp9),
+        Numpad0 => Ok(Key::Kp0),
+        Numpad1 => Ok(Key::Kp1),
+        Numpad2 => Ok(Key::Kp2),
+        Numpad3 => Ok(Key::Kp3),
+        Numpad4 => Ok(Key::Kp4),
+        Numpad5 => Ok(Key::Kp5),
+        Numpad6 => Ok(Key::Kp6),
+        Numpad7 => Ok(Key::Kp7),
+        Numpad8 => Ok(Key::Kp8),
+        Numpad9 => Ok(Key::Kp9),
 
-        VirtualKeyCode::Key0 => Ok(Key::Num0),
-        VirtualKeyCode::Key1 => Ok(Key::Num1),
-        VirtualKeyCode::Key2 => Ok(Key::Num2),
-        VirtualKeyCode::Key3 => Ok(Key::Num3),
-        VirtualKeyCode::Key4 => Ok(Key::Num4),
-        VirtualKeyCode::Key5 => Ok(Key::Num5),
-        VirtualKeyCode::Key6 => Ok(Key::Num6),
-        VirtualKeyCode::Key7 => Ok(Key::Num7),
-        VirtualKeyCode::Key8 => Ok(Key::Num8),
-        VirtualKeyCode::Key9 => Ok(Key::Num9),
+        Key0 => Ok(Key::Num0),
+        Key1 => Ok(Key::Num1),
+        Key2 => Ok(Key::Num2),
+        Key3 => Ok(Key::Num3),
+        Key4 => Ok(Key::Num4),
+        Key5 => Ok(Key::Num5),
+        Key6 => Ok(Key::Num6),
+        Key7 => Ok(Key::Num7),
+        Key8 => Ok(Key::Num8),
+        Key9 => Ok(Key::Num9),
 
-        VirtualKeyCode::Return => Ok(Key::Enter),
-        VirtualKeyCode::Space => Ok(Key::Space),
-        VirtualKeyCode::Escape => Ok(Key::Escape),
-        VirtualKeyCode::Equals => Ok(Key::Equal),
-        VirtualKeyCode::Minus => Ok(Key::Minus),
-        VirtualKeyCode::Back => Ok(Key::Backspace),
-        VirtualKeyCode::PageDown => Ok(Key::PageDown),
-        VirtualKeyCode::PageUp => Ok(Key::PageUp),
+        Return => Ok(Key::Enter),
+        Space => Ok(Key::Space),
+        Escape => Ok(Key::Escape),
+        Equals => Ok(Key::Equal),
+        Minus => Ok(Key::Minus),
+        Back => Ok(Key::Backspace),
+        PageDown => Ok(Key::PageDown),
+        PageUp => Ok(Key::PageUp),
 
-        VirtualKeyCode::Insert => Ok(Key::Insert),
-        VirtualKeyCode::Home => Ok(Key::Home),
-        VirtualKeyCode::Delete => Ok(Key::Delete),
-        VirtualKeyCode::End => Ok(Key::End),
+        Insert => Ok(Key::Insert),
+        Home => Ok(Key::Home),
+        Delete => Ok(Key::Delete),
+        End => Ok(Key::End),
 
-        VirtualKeyCode::Left => Ok(Key::Left),
-        VirtualKeyCode::Up => Ok(Key::Up),
-        VirtualKeyCode::Right => Ok(Key::Right),
-        VirtualKeyCode::Down => Ok(Key::Down),
+        Left => Ok(Key::Left),
+        Up => Ok(Key::Up),
+        Right => Ok(Key::Right),
+        Down => Ok(Key::Down),
 
-        VirtualKeyCode::LShift => Ok(Key::LeftShift),
-        VirtualKeyCode::LControl => Ok(Key::LeftControl),
-        VirtualKeyCode::LAlt => Ok(Key::LeftAlt),
-        VirtualKeyCode::LWin => Ok(Key::LeftSuper),
-        VirtualKeyCode::RShift => Ok(Key::RightShift),
-        VirtualKeyCode::RControl => Ok(Key::RightControl),
-        VirtualKeyCode::RAlt => Ok(Key::RightAlt),
-        VirtualKeyCode::RWin => Ok(Key::RightSuper),
```

Comment:
These changes look mostly unrelated, would've been nice to make a different commit for them.